packages/ag-charts-enterprise/src/features/annotations/annotationsStateMachine.ts

```diff
@@ -199,6 +199,18 @@ export class AnnotationsStateMachine extends StateMachine<States, AnnotationType
             ctx.update();
         };
 
+        const actionSaveText = ({ textInputValue }: { textInputValue?: string }) => {
+            const datum = ctx.datum(this.active!);
+            if (textInputValue != null && textInputValue.length > 0) {
```

Comment:
Would clearing the text (i.e. empty string) delete the annotation entirely? This seems undesirable. I would expect the annotation to only be removed when the trashcan icon is clicked.

Reply:
Yeah, that is the requirement from product. Previously only pressing enter would delete it, this change makes this the same for clicking off as well.